chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -197,7 +197,11 @@ def find_node_boot_mac_addresses(node, admin_data_net)
           hostname mnode.name
           macaddress mac_list[i]
           if admin_mac_addresses.include?(mac_list[i])
-            ipaddress admin_data_net.address unless admin_data_net.nil?
+            if admin_data_net.nil?
+              ipaddress mnode[:ipaddress]
+            else
+              ipaddress admin_data_net.address unless admin_data_net.nil?
```

Comment:
It's unneeded, yes, I just cherry-picked the commit that had this issue (the commit fixing this is fixing many other things, so let me add another commit just for this).